recipe_scrapers/keukenliefdenl.py

```diff
@@ -0,0 +1,58 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class KeukenLiefdeNL(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "keukenliefde.nl"
+
+    def author(self):
+        return self.soup.find("meta", {"name": "author"}).get("content")
+
+    def title(self):
+        return self.soup.find("meta", {"property": "og:title"}).get("content")
+
+    def category(self):
+        return self.soup.find(
+            "div", {"class": "article-meta-item sp gerecht"}
+        ).getText()
+
+    def total_time(self):
+        return get_minutes(
+            self.soup.find("div", {"class": "article-meta-item sp tijd"}).getText()
+        )
+
+    def yields(self):
+        return get_yields(
+            self.soup.find("div", {"class": "article-meta-item sp aantal"}).getText()
+        )
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = self.soup.find(
+            "div", {"id": "clipboard-ingredients"}
+        ).findChildren("li")
+
+        return [normalize_string(i.get_text()) for i in ingredients]
+
+    def instructions(self):
+        # Old recipes are written in paragraphs, new ones are a list.
+        paragraphs = self.soup.find("div", {"class": "preparation"}).findChildren("p")
+        listitems = self.soup.find("div", {"class": "preparation"}).findChildren("li")
+
+        instructions = [normalize_string(p.get_text()) for p in paragraphs] + [
+            normalize_string(i.get_text()) for i in listitems
+        ]
+
+        while "" in instructions:
+            instructions.remove("")
```

Comment:
A small oddity of the `while` loop here is that it'll restart checking the list from the beginning each time it finds (and then removes) an empty string.

Having it separated from the initial `instructions` variable declaration does make it clearer than we're filtering the results, though.

To combine those, here's a possible alternative approach:

```suggestion
        instructions = [
            normalize_string(item.get_text()) for item in paragraphs + listitems
        ]
        instructions = [
            instruction for instruction in instructions if instruction
        ]
```

It seems very repetitive, so I'm not sure if it's ideal.  Maybe the `if` condition could be placed directly into the former list comprehension.  I don't know whether Python's clever enough to avoid calling `normalize_string` twice if the filter (`if`) clause is identical to the emitted values.

Reply:
Thanks for you comments, I'm not a python developer, so it was already kind of hard to figure out how to do this ":-) 

I will check this later to see if I can update the code.

Reply:
That's totally fine, take your time :)  Thanks a lot for contributing!  And feel free to ask any questions.